packages/ListBox/src/components/Option/Option.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import { getA11yAttributesForOption } from "../../helpers/DOMAttributes";
+import { isOptionVisible, isOptionSelected, handleClickOption } from "../Options/helpers/options";
+import { OptionStyled } from "./Option.styles";
+
+const propTypes = {
+  /** String, number or JSX content */
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+
+  /** Describe if the option is selected or not */
+  isSelected: PropTypes.bool,
+
+  /** Describe if the option is enable or not */
+  isDisabled: PropTypes.bool,
+
+  /** Describe if the option is hidden or not */
+  isHidden: PropTypes.bool,
```

Comment:
Do we still need this? Wouldn't the option simply be filtered out from the app if it wasn't needed?

Reply:
Mmmm... no entirely sure if I want to remove it or keep it, I think for 0.5.0 version I will keep.